docs/content/configuration/virtualserver-and-virtualserverroute-resources.md

```diff
@@ -95,6 +96,28 @@ basedOn: scheme
 |``basedOn`` | The attribute of a request that NGINX will evaluate to send a redirect. The allowed values are ``scheme`` (the scheme of the request) or ``x-forwarded-proto`` (the ``X-Forwarded-Proto`` header of the request). The default is ``scheme``. | ``string`` | No | ### VirtualServer.Policy |
 {{% /table %}}
 
+### VirtualServer.TLS.CertManager
+
+The certmanager field configures the provisioning of automated certificate resources using certmanager for a VirtualServer (Please note that ACME Issuers are not yet supported):
+```yaml
+enable: true
+code: 301
+basedOn: scheme
+```
+
+{{% table %}}
+|Field | Description | Type | Required |
+| ---| ---| ---| --- |
+|``issuer`` |  the name of an Issuer to acquire the certificate required for this VirtualServer. The Issuer must be in the same namespace as the VirtualServer resource. | ``string`` | No |
+|``cluster-issuer`` | The name of a ClusterIssuer to acquire the Certificate required for this VirtualServer. It does not matter which namespace your VirtualServer resides, as ClusterIssuers are non-namespaced resources. | ``string`` | No |
```

Comment:
Are there precedence rules if `issuer` and `clusterissuer` are provided?

Reply:
If both are provided there will be a validation error - they are mutually exclusive. I'll add that.